docs/reference/remap/parse_json.cue

```diff
@@ -0,0 +1,46 @@
+package metadata
+
+remap: functions: parse_json: {
+	arguments: [
+		{
+			name:        "value"
+			description: "The text representation of the JSON to parse."
+			required:    true
+			type: ["string"]
+		},
+	]
+	return: ["boolean", "integer", "float", "string", "map", "array", "null"]
+	category: "parse"
+	description: #"""
+		Returns an `object` whose text representation is a JSON
+		payload in `string` form.
+
+		`string` must be the string representation of a JSON
+		payload. Otherwise, an `ParseError` will be raised.
+		"""#
+	examples: [
+		{
+			title: "Success"
+			input: {
+				message: #"{"key": "val"}"#
+			}
+			source: #"""
+				message = del(.message)
+				. = parse_json(message)
+				"""#
+			output: {
+				key: "val"
+			}
+		},
+		{
+			title: "Error"
+			input: {
+				message: "malformed"
```

Comment:
I believe this is valid? Primitive values are valid JSON. Something like `{ malformed }` would be invalid though.

Reply:
Hmm, although I guess it would have to be `"\"malformed\""` to be considered a string? So maybe this example doesn't work after all 😄

Reply:
It doesn't, but that isn't obvious. I'll update it with some obviously incorrect JSON.